app/presenters/notices/setup/prepare-return-forms.presenter.js

```diff
@@ -43,15 +50,17 @@ function _address() {
     addressLine5: 'United Kingdom'
   }
 }
-/*
- * {{ 'Water abstraction daily return ' if frequency === 'day' }}
- * {{ 'Water abstraction weekly return ' if frequency === 'week' }}
- * {{ 'Water abstraction monthly return ' if frequency === 'month' }}
- * {{ 'Water abstraction quarterly return ' if frequency === 'quarter' }}
- * {{ 'Water abstraction yearly return ' if frequency === 'year' }}
- */
-function _title() {
-  return 'Water abstraction daily return'
+
+function _pageTitle(returnsFrequency) {
+  const mapper = {
+    day: 'daily',
+    month: 'monthly',
+    quarter: 'quarterly',
+    week: 'weekly',
+    year: 'yearly'
+  }
```

Comment:
This already exists in `app/lib/static-lookups.lib.js`. See `returnRequirementFrequencies`.